src/probnum/diffeq/perturbedsolvers/_perturbedstepsolver.py

```diff
@@ -0,0 +1,110 @@
+"""ODE solver as proposed by Abdulle and Garegnani.
+
+References
+----------
+.. [1] https://arxiv.org/abs/1801.01340
+"""
+from typing import Callable, Optional
+
+import numpy as np
+
+from probnum import _randomvariablelist, diffeq, randvars, utils
+from probnum.diffeq.wrappedscipysolver import WrappedScipyRungeKutta
+from probnum.type import FloatArgType, RandomStateArgType
+
+
+class PerturbedStepSolver(diffeq.ODESolver):
+
+    """ODE-Solver based on Abdulle and Garegnani[1]_. Perturbs the steps accordingly and
```

Comment:
the reference [1] cannot be found by sphinx. this is why the docs fail. do you know how to fix htis?